runtime/compiler/compile/J9SymbolReferenceTable.cpp

```diff
@@ -2322,6 +2322,20 @@ J9::SymbolReferenceTable::findOrCreateProfilingBufferEndSymbolRef()
    return element(profilingBufferEndSymbol);
    }
 
+TR::SymbolReference *
+J9::SymbolReferenceTable::findOrCreateTempSlotSymbolRef()
+   {
+   if (!element(tempSlotSymbol))
+      {
+      TR_J9VMBase *fej9 = (TR_J9VMBase *)(fe());
+      TR::Symbol * sym = TR::RegisterMappedSymbol::createMethodMetaDataSymbol(trHeapMemory(), "tempSlot");
```

Comment:
This label is also potentially confusing in the lop with the other occurrences of temp slot - I think some thought needs to go into changing this to make it less confusing.

Reply:
Changed the label to something more explicit.